docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -49,6 +49,12 @@ OGC standards define outer ring vertices in counterclockwise order with inner
 ring (hole) vertices in clockwise order.
 
 Individual GeoJSON or WKT documents can override this parameter.
+
+// tag::geoshape-dateline-orientation[]
+Geoshapes that cross the international dateline (+/-180° longitude) must use a
```

Comment:
I think here we should explain when orientation has an effect, something on this lines:

 If the difference between the minimum longitude and the maximum longitude of the polygon is lower than 180 degrees, orientation has no effect. Otherwise the polygon is expected to have the same orientation as the one defined here. If the orientation is different, it is assume that the polygon crosses the international dateline (+/-180° longitude) and it is split around it.

wdyt?

Reply:
Note that is the polygon section there is an explanation. Maybe here we should have a pointer to that explanation and then we should try to rephrase the current explanation (which I thin is very obscure) with what I wrote above (which I hope is a bit clearer)